### ISSUE TITLE
Questions about loadSize

### ISSUE THREAD

**Issue by jikerWRN** [microsoft/S2R-DepthNet#9]
Hi, thanks for your work again!

I have some questions about loadSize, the rgb and depth image first resize to loadSize (192, 640) before feeding in the network.
```
scale_transform = transforms.Compose([transforms.Resize(self.size, Image.BICUBIC)])
img1 = scale_transform(img1)              ## RGB image
```
And resize to original size after network.
`pred_depth = torch.nn.functional.interpolate(pred_depth[-1], size=[depth_.size(1),depth_.size(2)], mode='bilinear',align_corners=True)`

I want to know why you downsample the image from (375, 1242) to (192, 640)? Is it just to reduce GPU memory?  Will it cause performance degradation due to loss of information?

**Comment by Xt-Chen** [comment]
On the one hand, reducing the image resolution is due to the limitation of GPU memory, on the other hand, for fair comparison with other methods, we follow the previous domain adaptation methods (T2Net/GASDA) to process the dataset. If you increase the image resolution, performance may be further improved.

**Comment by jikerWRN** [comment]
Thanks for your reply!
